gradle/dependencies.gradle

```diff
@@ -165,7 +165,7 @@ versions += [
   spotbugs: "4.8.0",
   zinc: "1.9.2",
   zookeeper: "3.8.3",
-  zstd: "1.5.5-6"
```

Comment:
We shouldn't bump dependencies in this PR. Let's have a different PR if we want to bump zstd.

Reply:
Ah! of course. This change missed my attention.